ember-cli-build.js

```diff
@@ -5,7 +5,7 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
     'ember-cli-babel': {
-      throwUnlessParallelizable: true,
```

Comment:
is ember-cli-typescript preventing this? 😱

Reply:
cc @chriskrycho I'd recommend setting up ember-cli-typescript's tests to ensure that we don't need to disable this prior to stable release.

Reply:
unsure, I need to re-rebase

Reply:
I'm not aware of anything we're doing that would prevent this, but agreed we should update our test suite to ensure it.

Reply:
Just published 2.0.0-beta.2 with a fix